x-pack/plugins/spaces/public/management/edit_space/enabled_features/enabled_features.tsx

```diff
@@ -8,7 +8,7 @@ import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiSpacer, EuiText, EuiTitle } from
 import { FormattedMessage, InjectedIntl } from '@kbn/i18n/react';
 import React, { Component, Fragment, ReactNode } from 'react';
 import { Capabilities } from 'src/core/public';
-import { Feature } from '../../../../../../../plugins/features/public';
+import { Feature } from '../../../../../../plugins/features/public';
 import { Space } from '../../../../common/model/space';
 import { getEnabledFeatures } from '../../lib/feature_utils';
 import { SectionPanel } from '../section_panel';
```

Comment:
__note:__ not related to this PR and isn't big deal at all, just wanted to check if it's known: noticed while testing that even if security plugin is disabled we still display `Want to secure access? Go to Roles` link that leads to nowhere. Wondering if we should check `securityLicense.showLinks` instead...

Reply:
Ah, I had fixed this in another place with https://github.com/elastic/kibana/pull/44829, but I missed the `<EnabledFeatures />` component. This is an easy fix, I'll update before merging.